src/include/txn_inline.h

```diff
@@ -2250,6 +2250,22 @@ __wt_txn_read_last(WT_SESSION_IMPL *session)
         __wt_txn_release_snapshot(session);
 }
 
+/*
+ * __wt_txn_read_last_check --
```

Comment:
This name doesn't seem very descriptive to me.  Maybe `__wt_txn_should_release_snapshot`?